### PR TITLE
Issue #DP-784 fix: UI Text changes and content filter issue fixes

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts [hunk 1]
```diff
@@ -235,8 +263,52 @@ export class MvcLibraryComponent implements OnInit {
...
+      return {
+        found: true,
+        parents: [...parents, this.generateParentLevel(tree, parents)]
+      };
+    }
+    let result = {
+      found: false,
+    };
+    if (tree.children) {
+      $.each(tree.children, (index, subtree) => {
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts:278]: is it `$` or `_`

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts [hunk 2]
```diff
@@ -197,15 +225,15 @@ export class MvcLibraryComponent implements OnInit {
...
       this.isFilterOpen = event.filterStatus;
-    } else {}
-  }
-
-  isResourceAdded(identifier): Boolean {
-    return _.includes(this.childNodes, identifier);
+    } else if (event.action === 'showAddedContent') {
+      this.showAddedContent = !event.status;
+      this.filterContentList();
+    } else { }
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts:236]: if there is no `else` then make `elseif` as else.?

Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts:236]: OK, I have removed `else` part, but we can't make `elseif` as `else` bcz condition needs to be check so..

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts (hunk 1 above)
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts:278]: it's `$` sign

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts [hunk 3]
```diff
@@ -235,8 +263,52 @@ export class MvcLibraryComponent implements OnInit {
...
+        found: true,
+        parents: [...parents, this.generateParentLevel(tree, parents)]
+      };
+    }
+    let result = {
+      found: false,
+    };
+    if (tree.children) {
+      $.each(tree.children, (index, subtree) => {
+        const maybeParents = $.merge([], parents);
```
[the remark this answers is not in the document]
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.ts:279]: As mentioned above, it's `$` sign

**Comment by vaibhavbhuva** [comment]
@venkateshwarans Resolved.

**Pull request merged**